review request (/cc @user) does not work

The `/cc` and `/uncc @user` slash commands don't update the reviewers:
```
{"Branch":"master","Clone":"...","ID":"92162432","Link":"...","Name":"...","Namespace":"...","Webhook":"issue_comment","author":"masseybradley","level":"info","msg":"Adding reviewer(s) to ...: [masseybradley]","org":"...","plugin":"assign","pr":...,"repo":"...","time":"2020-05-05T23:29:24Z","url":"..."}
panic: implement me

goroutine 234 [running]:
github.com/jenkins-x/lighthouse/pkg/scmprovider.(*Client).RequestReview(0xc000954340, 0xc0006c0a10, 0x9, 0xc0006c0a00, 0xa, 0x35a, 0xc0006e0bf0, 0x1, 0x1, 0x1943020, ...)
	/workspace/source/pkg/scmprovider/reviews.go:35 +0x39
github.com/jenkins-x/lighthouse/pkg/plugins/assign.handle(0xc00022d720, 0x1, 0xc0003c8c60)
	/workspace/source/pkg/plugins/assign/assign.go:149 +0x6ed
github.com/jenkins-x/lighthouse/pkg/plugins/assign.handleGenericComment(0x1f9f640, 0xc00032ef00, 0xc000954340, 0x1f4b380, 0xc000924630, 0x1f727a0, 0xc00013e850, 0x1f84be0, 0xc00051b560, 0x0, ...)
	/workspace/source/pkg/plugins/assign/assign.go:82 +0x14b
github.com/jenkins-x/lighthouse/pkg/webhook.(*Server).handleGenericComment.func1(0xc00098d740, 0xc00095e700, 0xc0000df900, 0xc0008e3ee0, 0x6, 0x1d22f10)
	/workspace/source/pkg/webhook/events.go:137 +0x28d
created by github.com/jenkins-x/lighthouse/pkg/webhook.(*Server).handleGenericComment
	/workspace/source/pkg/webhook/events.go:129 +0x151
```

Using version: `gcr.io/jenkinsxio/lighthouse-webhooks:0.0.568`

--- a/pkg/scmprovider/reviews.go
+++ b/pkg/scmprovider/reviews.go
@@ -4,6 +4,7 @@ import (
 	"context"
 
 	"github.com/jenkins-x/go-scm/scm"
+	"github.com/pkg/errors"
 )
 
 // ListReviews list the reviews
@@ -32,10 +33,16 @@ func (c *Client) ListReviews(owner, repo string, number int) ([]*scm.Review, err
 
 // RequestReview requests a review
 func (c *Client) RequestReview(org, repo string, number int, logins []string) error {
-	panic("implement me")
+	ctx := context.Background()
+	fullName := c.repositoryName(org, repo)
+	_, err := c.client.PullRequests.RequestReview(ctx, fullName, number, logins)
+	return errors.Wrapf(err, "requesting review from %s", logins)
 }
 
 // UnrequestReview unrequest a review
 func (c *Client) UnrequestReview(org, repo string, number int, logins []string) error {
-	panic("implement me")
+	ctx := context.Background()
+	fullName := c.repositoryName(org, repo)
+	_, err := c.client.PullRequests.UnrequestReview(ctx, fullName, number, logins)
+	return errors.Wrapf(err, "unrequesting review from %s", logins)
 }

